languages/strings.php

```diff
@@ -2,6 +2,7 @@
 exit;
 __( 'VAT', 'woocommerce-pdf-invoices-packing-slips' );
 __( 'Tax rate', 'woocommerce-pdf-invoices-packing-slips' );
+__( 'Payment Date:', 'woocommerce-pdf-invoices-packing-slips' );
```

Comment:
I think the `:` is not necessary, which means that the template strings are wrong too and need to be fixed. Also `Date` could be `date`.

Reply:
@alexmigf done in [80c9622](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/683/commits/80c9622819d399257b66d3f836a9c162449d64c0)

Reply:
I don't think this strings are for templates: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/issues/442